kafkaesque-core/src/main/java/in/rcard/kafkaesque/consumer/KafkaesqueConsumerConfigMapper.java

```diff
@@ -0,0 +1,31 @@
+package in.rcard.kafkaesque.consumer;
+
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+
+public class KafkaesqueConsumerConfigMapper {
+
+    public static Map toConsumerProperties(KafkaesqueConsumerConfig kConsumerConfig) {
```

Comment:
You don't need the `KafkaesqueConsumerConfigMapper` type at all. You can move the method `toConsumerProperties` directly in the `KafkaesqueConsumerConfig` class.